components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAttemptsByType/src/IsAnalyticsAttemptsByType.jsx

```diff
@@ -403,8 +403,8 @@ class IsAnalyticsAttemptsByType extends Widget {
             paddingRight: width * 0.05,
             paddingTop: height * 0.05,
             paddingBottom: height * 0.05,
-            height,
-            width,
+            display: 'flex',
+            'flex-direction': 'column',
```

Comment:
Why set flex direction? IMO we don't need it.

Reply:
If we use `display: 'flex'` only, the `div`'s inside will arranged side-by-side. But the requirement is to arrange them one under the other. For that purpose, `'flex-direction': 'column'` is used.

See the following link:
https://www.w3schools.com/css/css3_flexbox.asp [The flex-direction Property]

Reply:
@erangatl Shall we use `width: '100%', height: '100%'` here?